test/page.spec.js

```diff
@@ -534,7 +534,7 @@ module.exports.describe = function({testRunner, expect, headless, playwright, FF
       const result = await page.content();
       expect(result).toBe(expectedOutput);
     });
-    it.skip(FFOX || WEBKIT)('should not confuse with previous navigation', async({page, server}) => {
+    it('should not confuse with previous navigation', async({page, server}) => {
```

Comment:
This reliably does not work with .repeat(100) and -j 10 on my mac. I spent two hours trying to make it work on WebKit, and it seems like we don't always get load event after setContent. I'll keep digging.